pulp_rpm/app/models/repository.py

```diff
@@ -212,11 +206,8 @@ class RpmRepository(Repository):
     metadata_signing_service = models.ForeignKey(
         AsciiArmoredDetachedSigningService, on_delete=models.SET_NULL, null=True
     )
-    last_sync_revision_number = models.CharField(max_length=20, null=True)
```

Comment:
I believe you should write a data migration and move out this information into `last_sync_details` for existing repos